### ISSUE TITLE
desktop sharing stopped working... 

### ISSUE THREAD

**Issue by rbeldin** [tieto/sipe#144]
Some update in the past couple of weeks seems to have caused desktop sharing on Ubuntu 16.04 to stop working.  Sometimes, I will get a dialog that pops up that says something like: 

File /run/user/1000/sipe/
applicationsharing-3933-0x3965f68.remmina not found

If I look in /run/user/1000/sipe, it is empty.  I don't see any evidence in system logs that it was attempted to start. 
![screenshot_20170322_111500](https://cloud.githubusercontent.com/assets/12463449/24205628/448734de-0ef2-11e7-8f3c-50a132c30b23.png)

These are the components I have installed: 

```
 dpkg -l | grep rcgit+sipe | awk '{print $1,$2,$3}'
ii remmina 1.2.0~rcgit+sipe-0+201608170748~ubuntu16.04.1
ii remmina-common 1.2.0~rcgit+sipe-0+201608170748~ubuntu16.04.1
ii remmina-plugin-rdp 1.2.0~rcgit+sipe-0+201608170748~ubuntu16.04.1
ii remmina-plugin-vnc 1.2.0~rcgit+sipe-0+201608170748~ubuntu16.04.1

 dpkg -l | grep sipe+collab | awk '{print $1,$2,$3}'
ii gir1.2-farstream-0.2 0.2.8+sipe+collab-1+201701011925~ubuntu16.04.1
ii gstreamer0.10-nice:amd64 0.1.13+sipe+collab-201603090603+2222~ubuntu16.04.1
rc libfarstream-0.2-2:amd64 0.2.7~sipe+collab-0.2991+201512071752~ubuntu15.04.1
ii libfarstream-0.2-5:amd64 0.2.8+sipe+collab-1+201701011925~ubuntu16.04.1
ii libfarstream-0.2-dev 0.2.8+sipe+collab-1+201701011925~ubuntu16.04.1
```

I'm not sure how to isolate this further except by gathering logs from pidgin. 

What is the mechanism used to start remmina in this case?  Maybe that will help isolate this. 

Thanks, 

Rick

**Comment by xhaakon** [comment]
I can't reproduce this problem on latest 16.04.

> File /run/user/1000/sipe/applicationsharing-3933-0x3965f68.remmina not found

It works so that Sipe first establishes a connection with the presenter, then generates a `*.remmina` file in `$XDG_RUNTIME_DIR` and finally launches remmina with that file as an argument. If the file can't be written, remmina should not get started, so it looks as if your `*.remmina` were deleted in the meantime between the RDP client launch and the moment it tries to actually read the file.

I guess [Pidgin log](https://github.com/tieto/sipe/wiki/Debug-HOWTO#how-to-debug-a-malfunction-in-a-feature-that-should-work) could tell us more - there should be messages like "Written .remmina file ...", "Launching remmina: ...", or an error mentioned if `*.remmina` can't be saved.

Maybe check that you don't have multiple remmina instances running at the same time?

The attached `dpkg -l` output is incomplete. [Wiki](https://github.com/tieto/sipe/wiki/Debug-HOWTO#package-versions) has a command to get a full list of relevant packages.

**Comment by rbeldin** [comment]
Hmm... multiple remmina instances at one time?  That is entirely possible as I use remmina on a regular basis and may have remmina running at times.  

I'll post the versions and turn on logging.

**Comment by xhaakon** [comment]
> Hmm... multiple remmina instances at one time? That is entirely possible as I use remmina on a regular basis and may have remmina running at times.

I mean it's generally okay to have remmina already running when you accept a desktop sharing - the new instance launched from Pidgin should just pass the connection to the existing one and exit immediately. It's that I've seen it getting stuck from time to time - particularly after an upgrade has been made - so if you're having problems, I'd advise you to look at your process list and kill any running remminas, just in case.

**Comment by rbeldin** [comment]
I've got pidgin running with debug options to catch the issue the next time it happens.  

Here is the package list I have:

```
rbeldin@rbeldin-8460p:~$ dpkg --list | grep -e sipe -e pidgin -e farstream -e remmina -e nice -e freerdp | awk '/^ii/{print $2,$3}'
gir1.2-farstream-0.1 0.1.2-3ubuntu1
gir1.2-farstream-0.2 0.2.8+sipe+collab-1+201701011925~ubuntu16.04.1
gstreamer0.10-nice:amd64 0.1.13+sipe+collab-201603090603+2222~ubuntu16.04.1
gstreamer1.0-nice:amd64 0.1.14~sipe-0+201701270901~ubuntu16.04.1
libfarstream-0.1-0:amd64 0.1.2-3ubuntu1
libfarstream-0.1-dev 0.1.2-3ubuntu1
libfarstream-0.2-5:amd64 0.2.8+sipe+collab-1+201701011925~ubuntu16.04.1
libfarstream-0.2-dev 0.2.8+sipe+collab-1+201701011925~ubuntu16.04.1
libfreerdp-cache1.1:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-client1.1:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-client2:amd64 2.0.0~git20160503.1.f828595+dfsg1-0~201605121048~ubuntu16.04.1
libfreerdp-codec1.1:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-common1.1.0:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-core1.1:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-crypto1.1:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-gdi1.1:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-locale1.1:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-primitives1.1:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-server2:amd64 2.0.0~git20160503.1.f828595+dfsg1-0~201605121048~ubuntu16.04.1
libfreerdp-shadow2:amd64 2.0.0~git20160503.1.f828595+dfsg1-0~201605121048~ubuntu16.04.1
libfreerdp-utils1.1:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp2:amd64 2.0.0~git20160503.1.f828595+dfsg1-0~201605121048~ubuntu16.04.1
libnice10:amd64 0.1.14~sipe-0+201701270901~ubuntu16.04.1
libtelepathy-farstream-dev 0.6.2-1build1
libtelepathy-farstream3:amd64 0.6.2-1build1
pidgin 1:3.0.0~collab-1+201703131650~ubuntu16.04.1
pidgin-data 1:3.0.0~collab-1+201703131650~ubuntu16.04.1
pidgin-sipe 1.21.1+sipe-0+201703231431~ubuntu16.04.1
remmina 1.2.0~rcgit+sipe-0+201608170748~ubuntu16.04.1
remmina-common 1.2.0~rcgit+sipe-0+201608170748~ubuntu16.04.1
remmina-plugin-rdp 1.2.0~rcgit+sipe-0+201608170748~ubuntu16.04.1
remmina-plugin-vnc 1.2.0~rcgit+sipe-0+201608170748~ubuntu16.04.1
```

**Comment by rbeldin** [comment]
I have a complete log of a conference I attended where I tried to accept a shared desktop.  All that shows in the log regarding remmina is: 

```
 grep '(13:0' .pidgin-debug-2017-03-27T07\:28-04\:00.log  | grep remm
(13:02:55) sipe: Written .remmina file /run/user/1000/sipe/applicationsharing-5077-0x33aa788.remmina:
(13:02:55) sipe: Launching remmina: remmina -c /run/user/1000/sipe/applicationsharing-5077-0x33aa788.remmina
```

I could also see that /var/run/user/1000/sipe was modified at 13:02 and also later on when for someone reason remmina tried to startup again: 

drwx------ 2 rbeldin rbeldin  40 Mar 27 13:25 sipe

I looked in $HOME/.xsession-errors and can see this: 

```
 (remmina:13774): GLib-CRITICAL **: g_str_has_prefix: assertion 'str != NULL' failed
 QXcbConnection: XCB error: 3 (BadWindow), sequence: 18858, resource id: 31793237, major code: 3 (GetWindowAttributes), minor code: 0
 QXcbConnection: XCB error: 9 (BadDrawable), sequence: 18859, resource id: 31793237, major code: 14 (GetGeometry), minor code: 0
 QXcbConnection: XCB error: 3 (BadWindow), sequence: 22779, resource id: 100855561, major code: 15 (QueryTree), minor code: 0
 QXcbConnection: XCB error: 3 (BadWindow), sequence: 23298, resource id: 31904571, major code: 3 (GetWindowAttributes), minor code: 0
 QXcbConnection: XCB error: 9 (BadDrawable), sequence: 23299, resource id: 31904571, major code: 14 (GetGeometry), minor code: 0
 [13:25:11:170] [13774:24910] [ERROR][com.freerdp.core.transport] - BIO_read returned a system error 11: Resource temporarily unavailable
 [13:25:11:170] [13774:24910] [ERROR][com.freerdp.core.nego] - Protocol Security Negotiation Failure
 [13:25:11:170] [13774:24910] [ERROR][com.freerdp.core] - freerdp_set_last_error ERRCONNECT_SECURITY_NEGO_CONNECT_FAILED [0x2000C]
 [13:25:11:170] [13774:24910] [ERROR][com.freerdp.core.connection] - Error: protocol security negotiation or connection failure
 QXcbConnection: XCB error: 3 (BadWindow), sequence: 31731, resource id: 119655925, major code: 15 (QueryTree), minor code: 0
 Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.
 QXcbConnection: XCB error: 3 (BadWindow), sequence: 1899, resource id: 98736412, major code: 15 (QueryTree), minor code: 0
```

Not very accurate because .xession-errors has no timestamps...

**Comment by rbeldin** [comment]
During the meeting, remmina attempted to start several times, even as I was
viewing the presentation in skype for windows.  This time I got a timestamp in
.xsession-errors:

```
 (remmina:13774): GLib-CRITICAL **: g_str_has_prefix: assertion 'str != NULL'
failed
 [13:48:06:915] [13774:25678] [ERROR][com.freerdp.core.transport] - BIO_read returned a system error 11: Resource temporarily unavailable
 [13:48:06:915] [13774:25678] [ERROR][com.freerdp.core.nego] - Protocol Security Negotiation Failure
 [13:48:06:915] [13774:25678] [ERROR][com.freerdp.core] - freerdp_set_last_error ERRCONNECT_SECURITY_NEGO_CONNECT_FAILED [0x2000C]
 [13:48:06:915] [13774:25678] [ERROR][com.freerdp.core.connection] - Error: protocol security negotiation or connection failure
```
pidgin log:

```
*(13:48:06) sipe: Written .remmina file /run/user/1000/sipe/applicationsharing-5077-0x2ebe7b8.remmina:
*[remmina] name=sip:xxx.xxx@acme.com;gruu;opaque=app:conf:applicationsharing:id:3GMNZ5YZ
(Sipe desktop)
 protocol=RDP
 server=127.0.0.1:36527
 security=rdp
 scale=1
 aspectscale=1
 viewmode=1
 disableautoreconnect=1
*(13:48:06) sipe: Launching remmina: remmina -c /run/user/1000/sipe/applicationsharing-5077-0x2ebe7b8.remmina
 (13:48:06) sipe: RDP client launched.
 (13:48:06) sipe: process_input_message: removing CSeq 19
 (13:48:06) sipe: SIP transactions count:1 after removal
 (13:48:06) sipe: sip_sec_verify_signature: message is:<NTLM><078395BC><1319><SIP Communications Service><G9W8012.foo.bar.net><2E4BgDEEAa6A42iFF54m21ECtC9AAb7FE4x113Ex>< 538><NOTIFY><sip:xxx.xxx@acme.com;gruu;opaque=app:conf:focus:id:3GMNZ5YZ><0E140080 ><sip:xxx.yyy@acme.com><654772236><><><> signature to verify is:01000000fede6cdd7e04636b64000000
 (13:48:06) sipe: NTLM MAC(): Extented Session Security
 (13:48:06) sipe: NTLM MAC(): Key Exchange
 (13:48:06) sipe: NTLM calculated MAC: 01000000FEDE6CDD7E04636B64000000
 (13:48:06) sipe: sip_transport_input: signature of incoming message validated
 (13:48:06) sipe: process_input_message: msg->response(0),msg->method(NOTIFY)
 (13:48:06) sipe: send->process_incoming_notify
 (13:48:06) sipe: process_incoming_notify: subscription_state: active;expires=3600
 (13:48:06) sipe: sipe_dialog_find who='sip:xxx:xxx@acme.com;gruu;opaque=app:conf:chat:id:3GMNZ5YZ'
 (13:48:06) sipe: sipmsg_strip_headers: removing ms-user-logon-data
 (13:48:06) sipe: sipmsg_strip_headers: removing Max-Forwards
 (13:48:06) sipe: sipmsg_strip_headers: removing Authentication-Info
 (13:48:06) sipe: sipmsg_strip_headers: removing Content-Length
 (13:48:06) sipe: sipmsg_strip_headers: removing Content-Type
 (13:48:06) sipe: sipmsg_strip_headers: removing Event
 (13:48:06) sipe: sipmsg_strip_headers: removing subscription-state
 (13:48:06) sipe: sipmsg_strip_headers: removing Supported
 (13:48:06) sipe: NTLM MAC(): Extented Session Security
 (13:48:06) sipe: NTLM MAC(): Key Exchange
 (13:48:06) sipe: NTLM calculated MAC: 01000000A78A3E1B4A0864C364000000
*(13:48:06) sipe: RDP client has connected.
 (13:48:06) sipe: Written: 11
 (13:48:06) backend-fs2: farstream-send-codec-changed: codec: 127: application x-data clock:90000 channels:0
```

After the meeting finished, I got a popup from pidgin that I could not dismiss.   The OK button was unselectable.   Remmina had been started prior to the meeting:

![screenshot_20170327_141157](https://cloud.githubusercontent.com/assets/12463449/24370994/688bfe7e-12f7-11e7-9972-f60883f77858.png)

I have the full pidgin log that I can strip down and alter if that would be helpful but I don't want to post publicly.

**Comment by rbeldin** [comment]
It happened again today.  At the end of the conference, I got another request to see the presenters screen and looked quickly in /var/run/users/1000/sipe: 

```
cat applicationsharing-26908-0x41587c8.remmina
[remmina]
name=sip:john.doe@acme.com;gruu;opaque=app:conf:applicationsharing:id:NL3D2HRK 
(Sipe desktop)
protocol=RDP
server=127.0.0.1:46193
security=rdp
scale=1
aspectscale=1
viewmode=1
disableautoreconnect=1
```

So at some point it does get created but it never popped up.  xsession-errors shows this: 

```
 (remmina:13774): GLib-CRITICAL **: g_str_has_prefix: assertion 'str != NULL' 
failed
 [14:46:50:577] [13774:20608] [ERROR][com.freerdp.core.transport] - BIO_read 
returned a system error 11: Resource temporarily unavailable
 [14:46:50:578] [13774:20608] [ERROR][com.freerdp.core.nego] - Protocol Security 
Negotiation Failure
 [14:46:50:578] [13774:20608] [ERROR][com.freerdp.core] - freerdp_set_last_error 
ERRCONNECT_SECURITY_NEGO_CONNECT_FAILED [0x2000C]
 [14:46:50:578] [13774:20608] [ERROR][com.freerdp.core.connection] - Error: 
protocol security negotiation or connection failure
```

**Comment by xhaakon** [comment]
```
(remmina:13774): GLib-CRITICAL **: g_str_has_prefix: assertion 'str != NULL' 
failed
 [14:46:50:577] [13774:20608] [ERROR][com.freerdp.core.transport] - BIO_read 
returned a system error 11: Resource temporarily unavailable
 [14:46:50:578] [13774:20608] [ERROR][com.freerdp.core.nego] - Protocol Security 
Negotiation Failure
 [14:46:50:578] [13774:20608] [ERROR][com.freerdp.core] - freerdp_set_last_error 
ERRCONNECT_SECURITY_NEGO_CONNECT_FAILED [0x2000C]
 [14:46:50:578] [13774:20608] [ERROR][com.freerdp.core.connection] - Error: 
protocol security negotiation or connection failure
```

So it seems the problem is not with Sipe but with Remmina after all. I have seen similar random crashes after FreeRDP upgrade and the solution was to recompile Remmina against the updated library. I've forced a rebuild of freerdp and remmina on Launchpad so please update your packages from sipe-collab PPA and check if it is of any help.

I also noticed you have some packages installed that aren't needed by Sipe any longer. It shouldn't matter, but still you may consider removing them just in case. That is, if nothing else depends on them:

```
gstreamer0.10-nice:amd64 0.1.13+sipe+collab-201603090603+2222~ubuntu16.04.1
libfarstream-0.1-0:amd64 0.1.2-3ubuntu1
libfarstream-0.1-dev 0.1.2-3ubuntu1
gir1.2-farstream-0.1 0.1.2-3ubuntu1
libfreerdp-cache1.1:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-client1.1:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-codec1.1:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-common1.1.0:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-core1.1:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-crypto1.1:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-gdi1.1:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-locale1.1:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-primitives1.1:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-utils1.1:amd64 1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
```

**Comment by rbeldin** [comment]
Hi... 

I've removed the packages you suggested.  They seemed to be tied to an installation of vlc that I don't use any more, so no loss. 

I still had the issue during a meeting today with the latest packages, but I didn't have the debug enabled. 
Restarting to capture that. 

Here are the current packages I have installed: 

`gir1.2-farstream-0.2             0.2.8+sipe+collab-1+201701011925~ubuntu16.04.1
gstreamer1.0-nice:amd64                  0.1.14~sipe-0+201701270901~ubuntu16.04.1
libfarstream-0.2-5:amd64                 
0.2.8+sipe+collab-1+201701011925~ubuntu16.04.1
libfarstream-0.2-dev             0.2.8+sipe+collab-1+201701011925~ubuntu16.04.1
libfreerdp-client2:amd64                 
2.0.0~git20160503.1.f828595+dfsg1-0~201703290608~ubuntu16.04.1
libfreerdp-codec1.1:amd64                
1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-crypto1.1:amd64               
1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-locale1.1:amd64               
1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-primitives1.1:amd64           
1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp-server2:amd64                 
2.0.0~git20160503.1.f828595+dfsg1-0~201703290608~ubuntu16.04.1
libfreerdp-shadow2:amd64                 
2.0.0~git20160503.1.f828595+dfsg1-0~201703290608~ubuntu16.04.1
libfreerdp-utils1.1:amd64                
1.1.0~git20140921.1.440916e+dfsg1-5ubuntu1
libfreerdp2:amd64                
2.0.0~git20160503.1.f828595+dfsg1-0~201703290608~ubuntu16.04.1
libnice10:amd64                  0.1.14~sipe-0+201701270901~ubuntu16.04.1
libtelepathy-farstream-dev               0.6.2-1build1
libtelepathy-farstream3:amd64            0.6.2-1build1
pidgin           1:3.0.0~collab-1+201703251105~ubuntu16.04.1
pidgin-data              1:3.0.0~collab-1+201703251105~ubuntu16.04.1
pidgin-sipe              1.21.1+sipe-0+201703290652~ubuntu16.04.1
remmina                  1.2.0~rcgit+sipe-0+201703290633~ubuntu16.04.1
remmina-common           1.2.0~rcgit+sipe-0+201703290633~ubuntu16.04.1
remmina-plugin-rdp               1.2.0~rcgit+sipe-0+201703290633~ubuntu16.04.1
remmina-plugin-vnc               1.2.0~rcgit+sipe-0+201703290633~ubuntu16.04.1`

**Comment by magnusolsson80** [comment]
Hi.. Don't know if this will help. I have the same problem and I just ran pidgin in the terminal. I get the following error:

remmina: symbol lookup error: /usr/lib/x86_64-linux-gnu/remmina/plugins/remmina-plugin-rdp.so: undefined symbol: freerdp_channels_new

**Comment by rbeldin** [comment]
> remmina: symbol lookup error:
> /usr/lib/x86_64-linux-gnu/remmina/plugins/remmina-plugin-rdp.so: undefined
> symbol: freerdp_channels_new

I'm not getting this error at all.

-- 
Rick Beldin
Linux ERT
Tel: +1 770.343.0219  Email: rick.beldin@hpe.com
Physical: 5555 Windward Parkway West, Alpharetta GA 30004

**Comment by rbeldin** [comment]
I think this might be a config issue on my end.  I use the ufw firewall tool and have found that if I have the firewall enabled,  with the following settings, it won't start a desktop session: 

ufw status verbose
Status: active
Logging: off
Default: deny (incoming), allow (outgoing), deny (routed)
New profiles: skip

To                         Action      From
--                         ------      ----
Anywhere on lo             ALLOW IN    Anywhere
Anywhere (v6) on lo        ALLOW IN    Anywhere (v6)

I added the rule to allow anything on localhost assuming that is what remmina is listening to. 

remmina: 

sudo netstat -vantp | grep remmina
tcp        0      0 127.0.0.1:60468         127.0.0.1:34923         ESTABLISHED 20825/remmina

This would seem to be an ephemeral port. 

I see pidgin talking on this port: 

tcp        0      0 127.0.0.1:34923         127.0.0.1:60468         ESTABLISHED 4081/pidgin.orig


What ports would need to be open?

**Comment by xhaakon** [comment]
> What ports would need to be open?

@rbeldin Sipe doesn't specify any listen port for communication with Remmina and keeps it zero, so the system picks a random ephemeral port from `/proc/sys/net/ipv4/ip_local_port_range`. To me it gives a range from 32768 to 60999, but your mileage may vary.

> remmina: symbol lookup error: /usr/lib/x86_64-linux-gnu/remmina/plugins/remmina-plugin-rdp.so: undefined symbol: freerdp_channels_new

@magnusolsson80 this means that the versions of `remmina` and `libfreerdp2` you have installed are incompatible. Please check that you have the latest packages from [sipe-collab PPA](https://launchpad.net/~sipe-collab/+archive/ubuntu/ppa/+packages) and that you aren't by mistake using either FreeRDP or Remmina from another PPA, like [remmina-next](https://launchpad.net/~remmina-ppa-team/+archive/ubuntu/remmina-next/+packages).

**Comment by magnusolsson80** [comment]
> @magnusolsson80 this means that the versions of remmina and libfreerdp2 you have installed are incompatible. Please check that you have the latest packages from sipe-collab PPA and that you aren't by mistake using either FreeRDP or Remmina from another PPA, like remmina-next.

Hi. Sorry to say that I have only Remmina installed from the sipe-collab PPA.

$ dpkg -l | grep remmina
ii  remmina                                         1.2.0~rcgit+sipe-0+201610250728~ubuntu17.04.1 
ii  remmina-common                                  1.2.0~rcgit+sipe-0+201610250728~ubuntu17.04.1 
ii  remmina-plugin-rdp                              1.2.0~rcgit+sipe-0+201610250728~ubuntu17.04.1 
ii  remmina-plugin-vnc                              1.2.0~rcgit+sipe-0+201610250728~ubuntu17.04.1 

Update: I looked at the link that you supplied and the package versions do not match according to what I can see from the system when adding the ppa.

**Comment by xhaakon** [comment]
@magnusolsson80 Oh, you're using Ubuntu 17.04. sipe-collab didn't have a complete set of packages for that release. I've rebuilt the missing and out-of-date components, so please update from the PPA now and see if it helps.

https://launchpad.net/~sipe-collab/+archive/ubuntu/ppa/+packages?field.name_filter=&field.status_filter=published&field.series_filter=zesty

**Comment by magnusolsson80** [comment]
> @magnusolsson80 Oh, you're using Ubuntu 17.04. sipe-collab didn't have a complete set of packages for that release. I've rebuilt the missing and out-of-date components, so please update from the PPA now and see if it helps.

Thanks. The packages were upgraded.

**Comment by rbeldin** [comment]
@magnusolsson80 I've verified that disabling ufw prior to receiving a share request allows it to work again. I'm not sure what would be blocking it.  I loosed up some of the ports to the following: 

sudo ufw status verbose
Status: active
Logging: off
Default: deny (incoming), allow (outgoing), deny (routed)
New profiles: skip

To                         Action      From
--                         ------      ----
Anywhere on lo             ALLOW IN    Anywhere
3389/tcp                   ALLOW IN    Anywhere
5060/tcp                   ALLOW IN    Anywhere
6667/tcp                   ALLOW IN    Anywhere
3389/tcp (v6)              ALLOW IN    Anywhere (v6)
5060/tcp (v6)              ALLOW IN    Anywhere (v6)
6667/tcp (v6)              ALLOW IN    Anywhere (v6)

I'll see how this works.  Before, I only allowed lo .
